tests/testthat/test-dataset-entity.R

```diff
@@ -27,6 +27,26 @@ with_mock_crunch({
             '{"notes":"Ancillary information"}')
     })
 
+    test_that("Population setting", {
+        expect_equal(popSize(ds), 90000000)
+        expect_equal(popMagnitude(ds), 3)
+        expect_silent(setPopulation(ds, popSize(ds), popMagnitude(ds)))
```

Comment:
Use `expect_no_request` instead of `expect_silent` here: it's clearer what you're asserting.